pyresample/_spatial_mp.py

```diff
@@ -116,11 +125,14 @@ def __call__(self, data1, data2, inverse=False, radians=False,
                                           radians=radians, errcheck=errcheck)
 
 
-class Proj_MP(pyproj.Proj):
+class Proj_MP(BaseProj):
 
     def __init__(self, *args, **kwargs):
         self._args = args
         self._kwargs = kwargs
+        if not Proj.__init__ == object.__init__:
+            # pyproj-2.0+
```

Comment:
How about checking the pyproj version directly instead ?

Reply:
Agree, that's much clearer. How about 
```python
from distutils.version import StrictVersion
if StrictVersion(pyproj.__version__) >= StrictVersion('2.0.0')
    ...
```

Reply:
Hmm, maybe ``LooseVersion`` is better. Also works with strings like ``2.0.0-dev``

Reply:
Could probably do `pyproj.__version__ >= '2'`. Could also try importing CRS at the top of the module (except ImportError: CRS = None) and check if it is None here.

Reply:
Would you like me to change this in my pull request or save it for another pull request? That was @sfinkens change.

Reply:
Looking at this again, I'm confused why the if statement is here. We should always be calling the super class. Right? @sfinkens?

Reply:
In theory, yes. But the ``Proj`` class in pyproj < 2 does not implement its own ``__init__`` so that the ``super()`` call would be forwarded to ``object.__init__`` which does not accept any arguments

Reply:
@wroberts4 I guess do the version check with either of the methods described above.